pkg/backend/watch.go

```diff
@@ -108,8 +102,83 @@ func Watch(ctx context.Context, b Backend, stack Stack, op UpdateOperation,
 			display.PrintfWithWatchPrefix(time.Now(), "",
 				op.Opts.Display.Color.Colorize(colors.SpecImportant+"Update complete."+colors.Reset+"\n"))
 		}
-
 	}
 
 	return nil
 }
+
+func watchPaths(root string, paths []string) (chan string, func(), result.Result) {
```

Comment:
```suggestion
func watchPaths(root string, paths []string) (chan string, func(), error) {
```

This isn't ever returning results apart from errors

Reply:
Yep, updated accordingly.